### ISSUE TITLE
Can't build app with SDK 5.0.0 because of duplicated namespace

### ISSUE THREAD

**Issue by zsoltvilagos** [Adyen/adyen-android#1379]
**Describe the bug**

When building with dependency `com.adyen.checkout:action:5.0.0` I get this error:
```
repo % ./gradlew assembleDebug

> Task :app:processDebugMainManifest
[com.adyen.checkout:action:5.0.0] /Users/vilagoszs/.gradle/caches/transforms-3/d1932e2b8b11aec19369021a6cdf0827/transformed/action-5.0.0/AndroidManifest.xml Warning:
        Namespace 'com.adyen.checkout.action' is used in multiple modules and/or libraries: com.adyen.checkout:action:5.0.0, com.adyen.checkout:action-core:5.0.0. Please ensure that all modules and libraries have a unique namespace. For more information, See https://developer.android.com/studio/build/configure-app-module#set-namespace

> Task :app:checkDebugDuplicateClasses FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class com.adyen.checkout.action.BuildConfig found in modules action-5.0.0-runtime (com.adyen.checkout:action:5.0.0) and action-core-5.0.0-runtime (com.adyen.checkout:action-core:5.0.0)
     
     Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
```

**To Reproduce**
1. For the sake of the bug report, I generated a new Empty Activity project with Android Studio Giraffe 2023.3.1 Patch 2
2. Added `com.adyen.checkout:action:5.0.0` as implementation dependency to the `app` project
3. Compiled
4. Compile failed due to AndroidX browser (and I that others too) needs target SDK 34, so increased from 33.
5. Compiled again and got issue in Android Studio as well

**Expected behavior**
App builds :)

**Screenshots**
Not applicable.

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.0
 - Browser Chrome
 - Version -

**Smartphone (please complete the following information):**
 - Device: -
 - OS: -
 - Browser -
 - Version -

**Additional context**

Checked the `build.gradle` files in the repo `action-core` and `action` folders, they are indeed setting the same namespace.
* https://github.com/Adyen/adyen-android/blob/5.0.0/action-core/build.gradle#L22
* https://github.com/Adyen/adyen-android/blob/5.0.0/action/build.gradle#L22

**Comment by OscarSpruit** [comment]
Hi @zsoltvilagos, thank you for the detailed report! We will fix the issue asap and ship it with upcoming release (which should be very soon). Sorry for the inconvenience.

In the meantime, depending on your integration, it is possible to handle actions with the payment component itself: `cardComponent.handleAction(action, context)`.

**Comment by jreij** [comment]
Hi @zsoltvilagos, we just released [5.0.1](https://github.com/Adyen/adyen-android/releases/tag/5.0.1) which includes a fix for this issue, let us know if you still face any problems with it.

**Comment by zsoltvilagos** [comment]
> Hi @zsoltvilagos, we just released [5.0.1](https://github.com/Adyen/adyen-android/releases/tag/5.0.1) which includes a fix for this issue, let us know if you still face any problems with it.

Thank you for the heads-up. Already bumped, CI 🟢 .
